src/acbm/preprocessing.py

```diff
@@ -111,24 +109,49 @@ def nts_filter_by_region(
     # 1. Create a column in the PSU table with the region names
 
     # Dictionary of the regions in the NTS and how they are coded
+    # PSUGOR_B02ID but does not have values for 2021 and 2022
+    # region_dict = {
+    #     -10.0: "DEAD",
+    #     -9.0: "DNA",
+    #     -8.0: "NA",
+    #     1.0: "North East",
+    #     2.0: "North West",
+    #     3.0: "Yorkshire and the Humber",
+    #     4.0: "East Midlands",
+    #     5.0: "West Midlands",
+    #     6.0: "East of England",
+    #     7.0: "London",
+    #     8.0: "South East",
+    #     9.0: "South West",
+    #     10.0: "Wales",
+    #     11.0: "Scotland",
+    # }
+
+    # PSUStatsReg_B01ID but does not have values for 2021 and 2022
     region_dict = {
         -10.0: "DEAD",
         -9.0: "DNA",
         -8.0: "NA",
-        1.0: "North East",
-        2.0: "North West",
-        3.0: "Yorkshire and the Humber",
-        4.0: "East Midlands",
-        5.0: "West Midlands",
-        6.0: "East of England",
-        7.0: "London",
-        8.0: "South East",
+        1.0: "Northern, Metropolitan",
+        2.0: "Northern, Non-metropolitan",
+        3.0: "Yorkshire / Humberside, Metropolitan",
+        4.0: "Yorkshire / Humberside, Non-metropolitan",
+        5.0: "East Midlands",
+        6.0: "East Anglia",
+        7.0: "South East (excluding London Boroughs)",
+        8.0: "London Boroughs",
         9.0: "South West",
-        10.0: "Wales",
-        11.0: "Scotland",
+        10.0: "West Midlands, Metropolitan",
+        11.0: "West Midlands, Non-metropolitan",
+        12.0: "North West, Metropolitan",
+        13.0: "North West, Non-metropolitan",
+        14.0: "Wales",
+        15.0: "Scotland",
```

Comment:
Revised regions using field `"PSUStatsReg_B01ID"` as this provides values for 2021 and 2022.